__tests__/newrelic.tests.js

```diff
@@ -0,0 +1,42 @@
+const newrelic = require( '../src/newrelic/' );
+
+describe( 'Should fail if some environment variables are not present', () => {
```

Comment:
Minor: Ideally, we should wrap tests in a describe with the module name, i.e.

```
describe( 'src/newrelic', () => {
    describe( 'Should fail if some environment variables are not present', () => {
    ...
} );
```

For modules, with lots of exported values/methods/etc., it's also nice to break down further by the method-under-test:

```
describe( 'src/newrelic', () => {
    describe( 'newrelic()', () => {
        describe( 'should fail if env vars missing', () => {
            test( ... );
        } ); 
    } );
} );
```

Reply:
Oh, right, that would be nice. I'll work on it (and probably updating the others too)